tests/readme.md

```diff
@@ -0,0 +1,6 @@
+# Test Convention
+
+All test are decorated with either `testmode.standalone` or `testmode.integration`. The environment `DSS_TEST_MODE` 
+selects which type of tests to run. If the word "integration" is in the env var, then it runs the integration tests. If 
+the word "standalone" is in the env var, then it runs the standalone tests. `DSS_TEST_MODE` can contain both words, in 
+which case, both sets of tests are run. 
```

Comment:
Thank you for taking the time to keep the docs up to date!

I would add sth along the lines of: `Standalone tests may only use the fixture and storage buckets in each replica. They may not use any other cloud resources such as Elasticsearch instances, API gateway or Lambda functions.`

And I would add a reference to the respective Makefile targets.